HW3/homework3.py

```diff
@@ -0,0 +1,58 @@
+"""
+1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази.
+"""
+
+# print("I love Python "* 42)
```

Comment:
Please don't comment prints next time
```suggestion
print("I love Python " * 42)
```